docs/content/philosophy.md

```diff
@@ -78,4 +78,77 @@ dynamically into OPA remotely via APIs or through the local filesystem.
 
 OPA is a full-featured policy engine that offloads policy decisions from your
 service. You can think of it as a concierge for your service who can answer
-detailed questions on behalf of your users to meet their specific needs.
\ No newline at end of file
+detailed questions on behalf of your users to meet their specific needs.
+
+## How Does OPA Work? {#how-does-opa-work}
+
+> 📖 This section assumes you are familiar with the basics of OPA. We recommend
+> going through the [Introduction](..) before reading this section.
+
+OPA policies (written in Rego) make decisions based on hierarchical structured data.
```

Comment:
I'd be tempted to copy in the usual diagram and explanation from the Intro here.  Maybe as its own section.  That's "How OPA Works".  Then maybe this section is something titled maybe "OPA's information model" (though "information model" bothers me).

Reply:
Yep. What happened was that I couldn't find a good title. I didn't like the way the different options looked in the ToC, lol. I ended up settling on How OPA Works only to realize we needed to refer to the standard overview stuff.

Anyway, I think it's better in the latest version. I've put a link to the introduction that contains the standard overview and I'm fine with the new title ("The OPA Document Model"). Does that title work for you?

Reply:
Title seems good.